project/AppDependencies.scala

```diff
@@ -6,25 +6,25 @@ import sbt._
 object AppDependencies {
 
   val compile = Seq(
-    "uk.gov.hmrc"             %% "simple-reactivemongo"     % "7.31.0-play-27",
+    "uk.gov.hmrc"             %% "simple-reactivemongo"     % "8.0.0-play-28",
     "org.reactivemongo"       %% "reactivemongo-iteratees"  % "0.18.8", /* fix at 0.18.8 because that is what simple-reactivemongo 7.31.0-play-27 uses*/
     "org.reactivemongo"       %% "play2-reactivemongo"      % "0.18.8-play27", /* see above */
-    "uk.gov.hmrc"             %% "bootstrap-backend-play-27"% "3.3.0",
-    "uk.gov.hmrc"             %% "domain"                   % "5.10.0-play-27",
-    "uk.gov.hmrc"             %% "mongo-lock"               % "6.24.0-play-27",
-    "org.typelevel"           %% "cats-core"                % "2.3.1",
-    "com.enragedginger"       %% "akka-quartz-scheduler"    % "1.8.1-akka-2.5.x"
+    "uk.gov.hmrc"             %% "bootstrap-backend-play-28"% "4.2.0",
+    "uk.gov.hmrc"             %% "domain"                   % "5.11.0-play-27",
```

Comment:
I can see a few still using play 27? Is that working as expected?

Reply:
It does, for domain, it is the latest available.  

For play2-reactivemongo we need to stay with 0.18.8 because that is what simple-reactivemongo uses - reactivemongo has moved on to 1.0 with significant changes but not simple-reactivemongo and there is no 0.18.8-play-28 

hmrctest likewise has not been updated